sdk/search/search-documents/src/serviceModels.ts

```diff
@@ -2135,5 +2153,117 @@ export interface HnswParameters {
    */
   metric?: VectorSearchAlgorithmMetric;
 }
+
+/** Contains configuration options specific to the exhaustive KNN algorithm used during querying, which will perform brute-force search across the entire vector index. */
+export type ExhaustiveKnnVectorSearchAlgorithmConfiguration =
+  BaseVectorSearchAlgorithmConfiguration & {
+    /** Polymorphic discriminator, which specifies the different types this object can be */
+    kind: "exhaustiveKnn";
+    /** Contains the parameters specific to exhaustive KNN algorithm. */
+    parameters?: ExhaustiveKnnParameters;
+  };
+
+/** Contains the parameters specific to exhaustive KNN algorithm. */
+export interface ExhaustiveKnnParameters {
+  /** The similarity metric to use for vector comparisons. */
+  metric?: VectorSearchAlgorithmMetric;
+}
+
+/** A dictionary of index projection-specific configuration properties. Each name is the name of a specific property. Each value must be of a primitive type. */
+export interface SearchIndexerIndexProjectionsParameters {
+  /** Describes unknown properties. The value of an unknown property can be of "any" type. */
+  [property: string]: any;
```

Comment:
Can we type this as `Record<string, unknown>` instead?

Reply:
Done!